gems/pending/spec/postgres_ha_admin/failover_monitor_spec.rb

```diff
@@ -0,0 +1,141 @@
+require 'postgres_ha_admin/failover_monitor'
+require 'util/postgres_admin'
+
+describe PostgresHaAdmin::FailoverMonitor do
+  let(:db_yml) do
+    yml = double('DatabaseYml')
+    allow(yml).to receive(:pg_params_from_database_yml).and_return(:host => 'host.example.com', :user => 'root')
+    yml
+  end
+  let(:failover_db) { double('FailoverDatabases') }
+  let(:connection) do
+    conn = double("PGConnection")
+    allow(conn).to receive(:finish)
+    conn
+  end
+
+  let(:failover_monitor) do
+    expect(PostgresHaAdmin::DatabaseYml).to receive(:new).and_return(db_yml)
+    expect(PostgresHaAdmin::FailoverDatabases).to receive(:new).and_return(failover_db)
+    failover_instance = described_class.new('', '', @logger_file.path, 'test')
+    logger = double('Logger')
```

Comment:
If we have a logger file (created in the `before` hook) we don't need to be stubbing the logger calls.